showcase/README.md

```diff
@@ -153,26 +200,32 @@ $ cryptid controller add -c controlled.yml <UI_DID>
 
 ## Gateway
 
+Here we will show how to issue a gateway token to a cryptid address from a dummy gatekeeper. This example will show
```

Comment:
Probably need a line to explain what a dummy gatekeeper (or indeed a gatekeeper) is